.eslintrc.js

```diff
@@ -38,16 +49,15 @@ module.exports = {
         'tests/dummy/app/**'
       ],
       parserOptions: {
-        sourceType: 'script',
-        ecmaVersion: 2015
+        sourceType: 'script'
       },
       env: {
         browser: false,
         node: true
       },
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-        // add your custom rules and overrides for node files here
+        'node/no-unpublished-require': 'off'
```

Comment:
I've looked up this issue and still not sure why it is warning.  We have `glob`, `express`, etc in devDeps 🤔 

https://github.com/mysticatea/eslint-plugin-node/issues/47